README.md

```diff
@@ -188,6 +188,24 @@ This generates the following data to be sent.
 Published(destination='/topic/my_route_key_1', body='{"id": 1, "one": "Field One"}')
 Published(destination='/topic/my_route_key_2', body='{"id": 1, "two": "Field Two"}')
 ```
+## Stopping Publication
+
+The `@publish` decorator adds a named parameter called `stop` to the `save` method, which is useful when you need to update an instance of a decorated model and do not want this update to be published on the broker.
```

Comment:
Nice catch!
I would only suggest to rename `stop` to `send_to_queue` with default value `True`. This would bring a little more clearance to the intent of the argument.